independent-publisher-connectors/Informa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1976 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "INFORMA CALLS",
+    "description": "Informa D&B is the CESCE subsidiary that leads in providing Commercial, Financial, Sectorial, and Marketing Information of companies and entrepreneurs to enhance knowledge about clients and suppliers and minimize commercial risk.",
+    "termsOfService": "https://www.informa.es/",
+    "contact": {
+      "name": "Axazure",
+      "url": "https://www.axazure.com",
+      "email": "contact@axazure.com"
+    }
+  },
+  "host": "services.informa.es",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/get-predictive-companies": {
+      "get": {
+        "summary": "Get Company Information",
+        "description": "This service will enable the retrieval of a JSON document containing a list of companies, related according to the search criteria assigned.",
+        "operationId": "InformaGet-predictive",
+        "parameters": [
+          {
+            "name": "denominacion",
```

Comment:
Kindly correct the spelling "denominacion"

Reply:
I think I should use 'denominacion' which is what the API expects. Is in spanish.

Reply:
Kindly post your connector in English, this will be translated to other languages later.

Reply:
As per the public documentation link mentioned below, your connector, titles, descriptions, operations, summaries everything should be in English. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions